documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -115,7 +115,8 @@ For Kafka, the following aspects of a deployment can impact the resources you ne
 * The number of producers and consumers
 * The number of topics and partitions
 
-The values specified for resource requests are reserved and always available to a container.
+Resource requests set the resources a container needs to start.
```

Comment:
I would not use this. It is not completely true and it is misleading.

Reply:
removed